proto/cosmos/crypto/ed25519/keys.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+package cosmos.crypto.ed25519;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/crypto/keys/ed25519";
+
+// PubKey defines a ed25519 public key
+// Key is the compressed form of the pubkey. The first byte depends is a 0x02 byte
+// if the y-coordinate is the lexicographically largest of the two associated with
+// the x-coordinate. Otherwise the first byte is a 0x03.
+// This prefix is followed with the x-coordinate.
+message PubKey {
+    option (gogoproto.goproto_stringer) = false;
+
+    bytes key = 1;
+}
+
+// PrivKey defines a ed25519 private key.
+message PrivKey {
+    bytes key = 1;
+}
```

Comment:
Is this needed for `json` encoding? Should be marked with a warning of sort so that developers know that this should never be used other than in the place it is right now.

Reply:
hmm I'm not sure what you mean. We can't prevent developers marshalling this to JSON if they want.

Reply:
Resolved offline with Marko. This privKey is needed for the keyring.